scripts/dcos-windows-tox-tests.ps1

```diff
@@ -0,0 +1,43 @@
+$ErrorActionPreference = "Stop"
+
+
+function Confirm-DCOSToxRequirements {
+    # - Make sure the correct Python version is installed
+    $pythonVersion = python.exe --version
+    if($LASTEXITCODE) {
+        Throw "Failed to get the Python version"
+    }
+    if($pythonVersion -ne "Python 3.6.5") {
```

Comment:
nitpick: Would like to see the expected version in a variable and the error output to use the variable to make it easier to change things, but this is a very minor nice-to-have.